include/espressif/phy_info.h

```diff
@@ -0,0 +1,456 @@
+   https://github.com/esp8266/Arduino/blob/master/cores/esp8266/core_esp8266_phy.c#L29
+
+   Part of esp-open-rtos. Copyright (C) 2016 Angus Gratton,
+   BSD Licensed as described in the file LICENSE.
+ */
+#ifndef _ESPRESSIF_PHY_INFO_H
+#define _ESPRESSIF_PHY_INFO_H
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <stddef.h>
+
+/* CRYSTAL_FREQ_xx values as used by sdk_phy_info_t.crystal_freq */
+#define CRYSTAL_FREQ_40M 0
+#define CRYSTAL_FREQ_26M 1
+#define CRYSTAL_FREQ_24M 2
+
+/* SDIO_CONFIG_xx values as used by sdk_phy_info_t.sdio_config */
+#define SDIO_CONFIG_AUTO 0   /* Uses pin strapping to determine */
+#define SDIO_CONFIG_SDIOV1_1 /* Data output on negative edge */
+#define SDIO_CONFIG_SDIOV2_0 /* data output on positive edge */
+
+/* BT_COEXIST_CONFIG_xx values as used by sdk_phy_info_t.bt_coexist */
+/* No bluetooth */
+#define BT_COEXIST_CONFIG_NONE 0
+/* Coexistence configuration A:
+   GPIO 0  - WLAN_ACTIVE
+   GPIO 14 - BT_ACTIVE
+   GPIO 13 - BT_PRIORITY
+   GPIO 3  - ANT_SEL_BT
+*/
+#define BT_COEXIST_CONFIG_A 1
+/* No coexistence, but Bluetooth enabled?
+   Unsure how this works?
+ */
+#define BT_COEXIST_CONFIG_PRESENT 2
+/* Coexistence configuration B:
+   GPIO 0 - WLAN_ACTIVE
+   GPIO 14 - BT_PRIORITY
+   GPIO 13 - BT_ACTIVE
+   GPIO 3  - ANT_SEL_BT
+*/
+#define BT_COEXIST_CONFIG_B 3
+
+/* BT_COEXIST_PROTOCOL_xx values for coexistence protocol,
+   field sdk_phy_info_t.bt_coexist_protocol
+ */
+#define BT_COEXIST_PROTOCOL_WIFI_ONLY 0
+#define BT_COEXIST_PROTOCOL_BT_ONLY 1
+
+/* Coexistence is enabled, Bluetooth has its own antenna */
+#define BT_COEXIST_PROTOCOL_FLAG_SEPARATE_ANT 2
+/* Coexistence is enabled, Bluetooth shares WiFi antenna */
+#define BT_COEXIST_PROTOCOL_FLAG_SHARE_ANT 4
+
+/* Coexistence is enabled, use only BT_ACTIVE signal */
+#define BT_COEXIST_PROTOCOL_FLAG_BT_ACTIVE_ONLY 0
+/* Coexistence is enabled, use both BT_ACTIVE and BT_PROTOCOL signals */
+#define BT_COEXIST_PROTOCOL_FLAG_BT_ACTIVE_PRIORITY 1
+
+/* DUAL_ANT_CONFIG_xx values for dual antenna config,
+   field sdk_phy_info_t.dual_ant_config
+
+   (Not really clear how this feature works, if at all.)
+*/
+#define DUAL_ANT_CONFIG_NONE 0
+/* antenna diversity for WiFi, use GPIO0 + U0RXD (?) */
+#define DUAL_ANT_CONFIG_DUAL 1
+/* TX/RX switch for external PA & LNA: GPIO 0 high, GPIO 3 low during TX */
+#define DUAL_ANT_CONFIG_TX_GPIO0_HIGH_GPIO3_LOW
+/* TX/RX switch for external PA & LNA: GPIO 0 low, GPIO 3 high during TX */
+#define DUAL_ANT_CONFIG_TX_GPIO0_LOW_GPIO3_HIGH
+
+
+/* CRYSTAL_SLEEP_xx values used for sdk_phy_info_t.crystal_sleep
+ */
+#define CRYSTAL_SLEEP_OFF 0
+#define CRYSTAL_SLEEP_ON 1
+#define CRYSTAL_SLEEP_GPIO16 2
+#define CRYSTAL_SLEEP_GPIO2 3
+
+/* RF Stage 0 & 1 attenuation constants. Use for sdk_phy_info_t.lp_atten_stage01
+
+   @note These values have been tested and are confirmed to work as
+   expected - provided low_power_en is set then the signal is
+   attenuated as per this setting.
```

Comment:
Maybe it's just me, but I'd say it would be useful to know these details... I'm not sure where they would belong, but definitely could fit into commit message. By "usb dongle" you mean normal wifi dongle, and looking at RSSI it provides?

Well, I guess it's common comment about someone's doing RE - it would be nice to know as much as possible in case someone else will continue that work or will need to re-verify some moments. I of course know how hard is to record all those details... (So the point of comments is ... people looking, it's not useless ;-) ).